### PR TITLE
gh-87390: Add __unpacked__ attribute to types.GenericAlias

### PR DESCRIPTION
**Pull request by JelleZijlstra**
#87390

### CODE REVIEW

**Comment by JelleZijlstra** [comment]
cc @mrahtz

File: Doc/library/types.rst [hunk 1]
```diff
@@ -311,6 +311,25 @@ Standard names are defined for the following types:
    .. versionchanged:: 3.9.2
       This type can now be subclassed.
 
+   .. attribute:: __origin__
```
Comment (Fidget-Spinner) [inline Doc/library/types.rst]: This section is a duplicate of https://docs.python.org/3/library/stdtypes.html#special-attributes-of-genericalias-objects. Maybe we should just link to that instead?

**Review by Fidget-Spinner — COMMENTED** [review]
Docs need fixing up, otherwise code LGTM.

File: Doc/library/types.rst (hunk 1 above)
Comment (JelleZijlstra) [reply Doc/library/types.rst]: Oh thanks, I missed that! I'll add a section there instead.

Comment (JelleZijlstra) [reply Doc/library/types.rst]: Fixed in 81cbc83ebaa450d2a7232281a7239cbe969d5566

**Review by Fidget-Spinner — APPROVED** [review]

**Pull request merged**